packages/updatePrescriptionStatus/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "updatePrescriptionStatus",
+  "version": "1.0.0",
+  "description": "pdate Prescription Status",
+  "main": "updatePrescriptionStatus.js",
+  "author": "NHS Digital",
+  "license": "MIT",
+  "type": "module",
+  "scripts": {
+    "unit": "POWERTOOLS_DEV=true NODE_OPTIONS=--experimental-vm-modules jest --no-cache --coverage",
+    "lint": "eslint . --ext .ts --max-warnings 0 --fix",
+    "compile": "tsc",
+    "test": "npm run compile && npm run unit",
+    "check-licenses": "license-checker --failOn GPL --failOn LGPL --start ../.."
+  },
+  "dependencies": {
+    "@aws-lambda-powertools/commons": "^1.15.0",
+    "@aws-lambda-powertools/logger": "^1.18.0",
+    "@aws-sdk/client-dynamodb": "^3.525.0",
+    "@aws-sdk/util-dynamodb": "^3.525.0",
+    "@middy/core": "^5.2.4",
+    "@middy/input-output-logger": "^5.2.4",
+    "@nhs/fhir-middy-error-handler": "^1.0.3"
```

Comment:
these should be ^2.0.0 - see https://github.com/NHSDigital/prescriptionsforpatients/pull/719

Reply:
Packages have been upgraded.